Model/lib/wdk/model/questions/queries/geneQueries.xml

```diff
@@ -115,10 +115,12 @@
         <column name="project_id"/>
        <sql>
              <![CDATA[
+      select source_id,gene_source_id,project_id,matched_result from (
```

Comment:
Why use group by here instead of distinct?  I would typically only use group by if I am extracting info from the group like min/max/count.  Do you have reason to believe the group by would be faster?

Reply:
Also I wonder if we should make sure we're not adding duplicates in the genelist installer.  We could still put distinct here to make sure, but in Bob's case we have to process 2x the results.